business/products.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Providers of machine translation services
+---
+
+# Providers
+
+Machine translation **providers** offer ready-to-use machine translation systems. Typically the systems are provided as APIs, and also as applications or integrations.
+
+They vary by:
+
+- language support
+- accuracy
+- price
+- customization options
+- data confidentiality
+
+and other features, as well as underlying architecture.
+
+
+The advantage of using a machine translation provider is that the service is already working and can be integrated immediately. The alternative is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
```

Comment:
Let's change this to "An advantage" and "An alternative"...  To be more neutral and evergreen.